flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -969,6 +971,8 @@
 
 	public static final String DEFAULT_ZOOKEEPER_DIR_KEY = "/flink";
 
+	public static final String DEFAULT_ZOOKEEPER_NAMESPACE_KEY = "/default_ns";
```

Comment:
Sure, we can change that. I just thought that default might be a little generic.